### ISSUE TITLE
Problems with `typing_extensions`

### ISSUE THREAD

**Issue by arthurmloureiro** [Samreay/ChainConsumer#114]
Hi @Samreay 

In every env I tried to install chainconsumer I had this issue:
```
ImportError: cannot import name 'deprecated' from 'typing_extensions'
```

I fixed this previously by following this https://stackoverflow.com/questions/77074676/importerror-cannot-import-name-deprecated-from-typing-extensions

but that eventually lead to my entire environment to collapse...

**Comment by Samreay** [comment]
Hi Arthur, any chance I can get a larger stack trace from you? ChainConsumer doesn't using type_extensions, though I know scipy does, so the issue might come from scipy rather than ChainConsumer (or a different library, scipy is just a common culprit of issues haha)

**Comment by arthurmloureiro** [comment]
Hi @Samreay !

Yeah I think the error is potentially coming from something like that...
I just created a fresh environment with Python 3.10 and installed only chainconsumer. When loading it I found no issues.

It's clearly something that I had on my other environments that, when installing chainconsumer, it caused that error to appear (potentially other modules would also cause it to appear).

[A classic case of "open issue and problem is solved by itself" apologies and thanks for taking the time!]
